harness/determined/pytorch/_load.py

```diff
@@ -174,7 +180,7 @@ def load_trial_from_checkpoint_path(
     if not is_trainer:
```

Comment:
can get rid of the trial_in_main logs above this.

Reply:
Good catch.  It looks like they are dead code right now, and given that this already passed the full upstream CI suite, I'm going to land this as-is and make a PR against master deleting the dead code.